packages/cozy-client/src/helpers.spec.js

```diff
@@ -150,6 +151,83 @@ describe('generateWebLink', () => {
   })
 })
 
+fdescribe('deconstructWebLink', () => {
```

Comment:
```suggestion
describe('deconstructWebLink', () => {
```

Reply:
Fixed in https://github.com/cozy/cozy-client/compare/637711a2193c5077ee2ada1cb7eec31c843ee6fc..f14039dd1b109d01d61f354cbd854fdb7804edf8

Thanks